OSDOCS-6652 AWS shared private hosted zones (#64114)
openshift/openshift-docs: installing/installing_aws/installing-aws-vpc.adoc

* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.

> * If the existing VPC is in a different account than your cluster, you link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[shared your VPC] between accounts.

Reviewer: The original wording is good, but this suggestion might help with some ideas.
```suggestion
* If the existing VPC is owned by a different account than your cluster, you link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[shared the VPC] with your cluster account.
```
